accepted/0000-link-packages-to-source-and-build.md

```diff
@@ -0,0 +1,635 @@
+
+As of today, Google Cloud Build supports custom claims but neither GitLab Pipelines nor CircleCI offers this. [GitLab](https://gitlab.com/gitlab-org/gitlab/-/issues/356986) is planning to add support for a custom audience.
+
+The plan is to start with direct vendor support in Fulcio. Fulcio will need to be patched to produce certificates containing custom claims data from Circle CI, GitLab and Google Cloud Build.
+
+We will also work with OpenSSF and other package ecosystems adopting [Sigstore](https://www.sigstore.dev/) on an official claims set that any provider can implement. Once we have this each vendor could be added to Fulcio by simply specifying the standard OIDC client configuration.
+
+#### Comparison of OIDC ID token support across CI/CD systems
+
+| Name               | OIDC ID token | Custom audience | Comment                                                                                                                                                                |
+|--------------------|--------------|-----------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| GitHub Actions     | yes          | yes             | Uses “workflow run” as the subject                                                                                                 |
+| GitLab Pipelines   | yes          | no              | Subject is the project path. See [documentation](https://docs.gitlab.com/ee/ci/variables/predefined_variables.html).                                                                                                                   |
+| CircleCI           | yes          | no              | Uses [workload ID](https://circleci.com/docs/2.0/openid-connect-tokens/) that includes a notion of “user ID” in the Fsubject claim. There is a [feature request to make this configurable](https://circleci.canny.io/cloud-feature-requests/p/customizable-audience-claim-in-oidc-tokens) |
+| Google Cloud Build | yes          | yes             |                                                                                                                                                     |
+| Cirrus CI.                  | yes          | yes             |                                                                                                                                                     |
+| Bitbucket          | yes          | no              | Is it used for OSS?                                                                                                                                                    |
+| TravisCI           | no           | -               | Is it used for OSS?                                                                                                                                                    |
```

Comment:
Fixed, thanks for the context!

Reply:
What they currently offer is a highly limited number of "free credits" a month, whereas the typical definition of "used for OSS" is "unlimited usage on public repos", so I'm not sure it qualifies without some kind of asterisk.

Reply:
I agree it's worth adding an asterisk, because yes, it's a limited support due to those free credits situation. However in practice it works well, at least for my own OSS project, which is admittedly not very active.

Reply:
> 1 of those remaining 3 runs out of credits every month. It’s not sufficient.

This doesn't sound great! Will update this.

Reply:
> and 1 of those remaining 3 runs out of credits every month
Yeah, that doesn't sound like a good deal indeed. Let's make that asterisk in bold.